packages/mdc-radio/mdc-radio.scss

```diff
@@ -33,7 +33,7 @@
   @include mdc-radio-ink-color($mdc-radio-baseline-theme-color);
   @include mdc-radio-focus-indicator-color($mdc-radio-baseline-theme-color);
   @include mdc-ripple-surface;
-  @include mdc-ripple-color($mdc-radio-baseline-theme-color, $mdc-radio-ripple-opacity);
+  @include mdc-states($mdc-radio-baseline-theme-color);
```

Comment:
nit: Put this after all of the `mdc-ripple-*` mixins unless there is some magical states reason not to.

Reply:
Done.